marko.json

```diff
@@ -93,5 +93,25 @@
             "@*": "expression"
         }
     },
-    "<ebay-breadcrumb-item>": {}
+    "<ebay-breadcrumb-item>": {},
+    "<ebay-pagination>": {
+        "renderer": "./src/components/ebay-pagination/index.js",
+        "transformer": "./src/common/transformer/index.js",
+        "@class": "string",
+        "@aria-label-prev": "string",
+        "@aria-label-next": "string",
+        "@curr-text": "string",
```

Comment:
Just to note, I used different names for approximately the same attributes, so we should try to keep these consistent (I'm fine changing mine too). @ianmcburnie Do you prefer certain names for these a11y attributes?

Reply:
I referred to the old carousal while naming I think. I will take a look at your names and change them.

Reply:
If anything, it should be `current-text`.